modules/codegen/src/smithy4s/codegen/internals/CollisionAvoidance.scala

```diff
@@ -396,6 +396,10 @@ private[internals] object CollisionAvoidance {
     val timestamp_ = NameRef("smithy4s", "Timestamp")
     val document_ = NameRef("smithy4s", "Document")
     val uuid_ = NameRef("smithy4s", "UUID")
+    val localdate_ = NameRef("smithy4s", "LocalDate")
+    val localtime_ = NameRef("smithy4s", "LocalTime")
+    val duration_ = NameRef("java.time", "Duration")
+    val offsetdatetime_ = NameRef("java.time", "OffsetDateTime")
```

Comment:
I thought these were:

```suggestion
    val duration_ = NameRef("scala.concurrent.duration.", "Duration")
    val offsetdatetime_ = NameRef("smithy4s", "OffsetDateTime")
```